src/OpenTracing/Log/Fields.cs

```diff
@@ -0,0 +1,57 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="Fields.cs">
+//   Copyright 2017-2018 The OpenTracing Authors
+//   
+//   Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+//   in compliance with the License. You may obtain a copy of the License at
+//   
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+//   Unless required by applicable law or agreed to in writing, software distributed under the License
+//   is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+//   or implied. See the License for the specific language governing permissions and limitations under
+//   the License.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OpenTracing.Log
+{
+    /// <summary>
+    ///     The following log fields are recommended for instrumentors who are trying to capture more
+    ///     information about a logged event. Tracers may expose additional features based on these
+    ///     standardized data points.
+    ///     <see href="https://github.com/opentracing/specification/blob/master/semantic_conventions.md" />
+    /// </summary>
+    public static class Fields
```

Comment:
I know this is a deviation from Java but what sense does it make to have a namespace with just one Type? Why not keep using "LogFields" in the root namespace which is more discoverable and a more descriptive name?

Reply:
I normally wouldn't really care about how many types are in a namespace (it seems rather pedantic of a metric). If logically, the item belongs in a different 'folder' if you sorted/grouped things, it deserves its own namespace. I think the 'rule' to keep it high is because many people may overuse namespaces and not use them for grouping.

That said, I don't really want to argue style in OSS, so I will change per your suggestion as it makes little difference to me.

Reply:
I added this to #55 - I can live with whatever most people want. It's not a blocker :)

Reply:
resolved